EnrichStream/Dockerfile

```diff
@@ -1,58 +1,38 @@
 #
-#  Install dependencies, compile sources, build jars.
+#  Install dependencies, compile sources, build jar.
 #
 FROM maven:3.9.6-eclipse-temurin-17-alpine as build
 
 WORKDIR /app
 
-ARG MAVEN_ARGS
-
 COPY pom.xml ./
 COPY settings.xml ./
-# Copy in local repository
-COPY .m2/ /root/.m2/
+COPY .mvn .mvn
```

Comment:
maven automatically looks for a .mvn directory. this way we use the maven.config both locally and in docker instead of explicitly supplying the settings file path in the dockerfile